### ISSUE TITLE
event_display displays received events on an HTTP endpoint

### ISSUE THREAD

**Issue by xtreme-sameer-vohra** [knative/eventing-contrib#1674]
**Problem**
All examples using event_display on knative.dev site include instructions to check for received events via
```
kubectl logs --selector='serving.knative.dev/service=event-display' -c user-container
```
It would also be beneficial to be able to view events on the service's http endpoint via a `GET`. 
I naturally clicked the link once the service was deployed and incorrectly assumed it was broken until I followed through the rest of the example and realized that the output is in the logs.

**[Persona:](https://github.com/knative/eventing/blob/master/docs/personas.md)**
IHMO it applies to all consumers as event_display is used across the knative docs samples

**Exit Criteria**
curl EVENT_DISPLAY_URI | grep  "some message attr/value"

**Time Estimate (optional):**
I am not certain as I haven't looked at the code, however, I am naively assuming its 1-2 days at most. 

**Additional context (optional)**
Add any other context about the feature request here.

**Comment by antoineco** [comment]
@xtreme-sameer-vohra you probably want to use Sockeye instead then: https://github.com/n3wscott/sockeye/.

I'm not sure whether it would make sense to expose a web server in event_display since that would require a complete redesign. You need to persist events, purge them from time to time, use websocket to have something as dynamic as tailing logs, etc. and Sockeye already does all of that.

**Comment by xtreme-sameer-vohra** [comment]
Yes, something like `sockeye` would be great for someone new and getting started. I think the issue is that its less discoverable than `event-display` but has exactly the UX that would be beneficial for someone new to Knative.

**Comment by xtreme-sameer-vohra** [comment]
I am ok with closing this issue if this hasn't been raised before but just wanted to bring up my view as someone who is learning/new

**Comment by antoineco** [comment]
I fully agree with that, Sockeye is an incredibly useful tool. @n3wscott would that be reasonable to make it more visible in the docs, although the project is not under the Kn organization?
